src/index.html

```diff
@@ -15,8 +15,105 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
+    <header class="header">
+      <div class="container">
+        <div class="header__logo">
+          <a
+            href="index.html"
+            class="logo__link"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="logo__image"
+            />
+          </a>
+        </div>
+        <nav class="header__nav">
+          <ul class="menu__list">
```

Comment:
```suggestion
        <nav class="header__nav nav">
          <ul class="nav__list">
```

Let's create a nav block and use it for all elements inside nav